client/Log.cpp

```diff
@@ -85,6 +85,15 @@ std::string FormatPacket(const OP2Internal::Packet& packet)
 	return ss.str();
 }
 
+std::string FormatLoadAddress(void* value)
+{
+	return FormatHexadecimal(reinterpret_cast<std::uintptr_t>(value));
+}

```

Comment:
This function works generically for any address value, not just for module base addresses. I would recommend naming it `FormatAddress`.